game/graphics/renderer.cpp

```diff
@@ -626,6 +716,49 @@ void Renderer::drawFxaa(Encoder<CommandBuffer>& cmd) {
   cmd.draw(Resources::fsqVbo());
   }
 
+void Renderer::applyCmaa2(Tempest::Encoder<Tempest::CommandBuffer>& cmd)
+{
+  // TODO: initialization that is needed only on the first run. Make it run only in the first run
```

Comment:
TODO

Reply:
Is there something like a current frame number to determine if the current frame is the first one?